src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -414,8 +414,10 @@ public function testSetCatchExceptions()
         }
     }
 
-    public function testAsText()
+    public function testLegacyAsText()
     {
+        $this->iniSet(-1 & E_USER_DEPRECATED);
```

Comment:
Aren't you missing an argument saying it should set ``error_reporting`` to this value ?